packages/Takeover/src/Takeover.styles.js

```diff
@@ -3,6 +3,19 @@ import tokens from "@paprika/tokens";
 import stylers from "@paprika/stylers";
 import OriginalHeader from "./components/Header";
 
+export const FocusLock = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  &::before,
+  &::after {
+    content: "";
+    display: block;
+  }
+`;
+
```

Comment:
I _think_ these styles are correct, although the `modal` components styles also have a `flex-basis` style which I'm unsure is needed here - https://github.com/acl-services/paprika/blob/master/packages/Modal/src/Modal.styles.js#L48